src/main/java/com/zaxxer/hikari/metrics/micrometer/MicrometerMetricsTracker.java

```diff
@@ -2,33 +2,32 @@
 
 import com.zaxxer.hikari.metrics.IMetricsTracker;
 import com.zaxxer.hikari.metrics.PoolStats;
-import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.Gauge;
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Timer;
+import io.micrometer.core.instrument.stats.hist.Histogram;
 import io.micrometer.core.instrument.stats.quantile.WindowSketchQuantiles;
 
 import java.util.concurrent.TimeUnit;
 
-import static io.micrometer.core.instrument.stats.hist.CumulativeHistogram.buckets;
-import static io.micrometer.core.instrument.stats.hist.CumulativeHistogram.linear;
-
 public class MicrometerMetricsTracker implements IMetricsTracker
 {
    private static final String METRIC_CATEGORY = "pool";
-   private static final String METRIC_NAME_WAIT = "Wait";
-   private static final String METRIC_NAME_USAGE = "Usage";
-   private static final String METRIC_NAME_CONNECT = "ConnectionCreation";
-   private static final String METRIC_NAME_TIMEOUT_RATE = "ConnectionTimeoutRate";
-   private static final String METRIC_NAME_TOTAL_CONNECTIONS = "TotalConnections";
-   private static final String METRIC_NAME_IDLE_CONNECTIONS = "IdleConnections";
-   private static final String METRIC_NAME_ACTIVE_CONNECTIONS = "ActiveConnections";
-   private static final String METRIC_NAME_PENDING_CONNECTIONS = "PendingConnections";
+   private static final String METRIC_NAME_WAIT = "hikaricp.connections.acquired.duration";
+   private static final String METRIC_NAME_USAGE = "hikaricp.connections.usage.duration";
+   private static final String METRIC_NAME_CONNECT = "hikaricp.connections.creation.duration";
+
+   private static final String METRIC_NAME_TIMEOUT_RATE = "hikaricp.connections.timeout.total";
```

Comment:
Why ``.total``?  Maybe ``.rate`` to indicate the metric type (similar to ``.duration`` above).

Reply:
Same as above, drop the word "total". It is added by the Promtheus naming convention. Elsewhere, not a common suffix.

Reply:
> Why .total
@brettwooldridge Note the metric type is a `counter`, as opposed to a DropWizard `meter`  The counter merely keeps track of the total failures as a number that only goes up. 

Prometheus and other systems that gather the metric then can compare the rate of the increase to display the rate themselves. For example a 5 minutes rate in prometheus query would be `rate(hikaricp_connections_timeout_total[5m])` (micrometer reformats the dot separated metric with the underscore format for Prometheus)